database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,23 @@
+SET ROLE zalando_nakadi_data_owner;
+
+CREATE TABLE zn_data.storage (
+  st_id            VARCHAR(36) NOT NULL PRIMARY KEY,
+  st_type          VARCHAR(32) NOT NULL,
+  st_configuration JSONB       NOT NULL
+);
+
+
```

Comment:
You slightly alter the already used good old Zalando DB naming conventions.

Fields for `zn_data.storage` should have prefix `s_` (i.e. `s_type`). Fields for `zn_data.event_type` have prefix `et_`.

Following the same convention in `zn_data.timeline`, you have:
- `t_id`,
- `t_event_type_name`,
- `t_storage_id`,
- `t_topic`.